src/python/WMCore/WorkQueue/DataStructs/WorkQueueElementResult.py

```diff
@@ -23,6 +23,8 @@ def __init__(self, **kwargs):
                             sum([x['EventsWritten'] for x in self['Elements']]))
             self.setdefault('FilesProcessed',
                             sum([x['FilesProcessed'] for x in self['Elements']]))
+            self.setdefault('Jobs',
```

Comment:
We can't add blowupFactor here. That will be double blowup. :smile: But for correct calculation of how many jobs will be created, you are right we have to add the same blowup factor right before the display. (If we add here it will be calculated again in the LQ WorkQueueManager.


> And a silly question, what's the difference in this if-else? Is the ParentQueueElement the workqueue_inbox doc that is created for each workflow, and which spawns > 1 WQE?

ParentQueueElement is a global workqueue element for given local workqueue elements.
So global workqueue elements doesn't have ParentQueueElement